web/src/main/java/org/akaza/openclinica/controller/ImportController.java

```diff
@@ -162,14 +162,14 @@ public ImportController() {
         UserAccountBean userAccountBean = utilService.getUserAccountFromRequest(request);
         ArrayList<StudyUserRoleBean> userRoles = userAccountBean.getRoles();
 
+        if (!validateService.isStudyAvailable(studyOid)) {
+            return new ResponseEntity(ErrorConstants.ERR_STUDY_OID_NOT_AVAILABLE, HttpStatus.OK);
+        }
 
         if (!validateService.isStudyOidValid(studyOid)) {
-            return new ResponseEntity(ErrorConstants.ERR_STUDY_NOT_EXIST, HttpStatus.NOT_FOUND);
+            return new ResponseEntity(ErrorConstants.ERR_STUDY_NOT_EXIST, HttpStatus.OK);
```

Comment:
why are we returning 200 on error?

Reply:
On most of the errors in endpoints we return 200. Requirements are in the ticket.
https://jira.openclinica.com/browse/OC-10880